Tests/AlicerceTests/UIKit/UIImageTestCase.swift

```diff
@@ -20,4 +20,28 @@ final class UIImageTestCase: XCTestCase {
 
         XCTAssertEqual(mrMinderTemplate.renderingMode, .alwaysTemplate)
     }
+
+    func testConvenienceInit_WithBase64String_ShouldReturnUIImage() {
+        let mrMinder = imageFromFile(withName: "mr-minder", type: "png")
+
+        guard let pngImageData = UIImagePNGRepresentation(mrMinder) else {
+            XCTFail("Could not convert mr-minder image to PNG representation")
+            return
+        }
+
+        let base64String = pngImageData.base64EncodedString()
+
+        guard let image = UIImage(base64Encoded: base64String) else {
+            XCTFail("Could not init UIImage from base64 representation")
+            return
+        }
+
+        XCTAssertEqual(UIImagePNGRepresentation(image), pngImageData)
+    }
+
+    func testConvenienceInit_WithNoneBase64String_ShouldReturnNilFromInit() {
```

Comment:
Maybe also add a unit test with a valid Base64 string that isn't a valid image, such as `Ym9vbQ==`

Reply:
Good idea 👍 Added in 9a72ccca293fc35bcb8f1d3de73827bdfa607ca9